content/en/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -6,14 +6,8 @@ weight: 20
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
```

Comment:
You could replace this line rather than remove it:
```markdown
{{< feature-state for_k8s_version="v1.18" state="stable" >}}
```

Reply:
updated